tests/test_e2e.py

```diff
@@ -472,39 +516,55 @@ def test_b2c_acquire_token_by_auth_code(self):
                 # This won't work https://msidlab.com/api/user?usertype=b2c
             password="***"  # From https://aka.ms/GetLabUserSecret?Secret=msidlabb2c
         """
-        scopes = ["https://msidlabb2c.onmicrosoft.com/msaapp/user_impersonation"]
-        (self.app, ac, redirect_uri) = _get_app_and_auth_code(
-            "b876a048-55a5-4fc5-9403-f5d90cb1c852",
-            client_secret=self.get_lab_user_secret("MSIDLABB2C-MSAapp-AppSecret"),
+        self._test_acquire_token_by_auth_code(
             authority=self._build_b2c_authority("B2C_1_SignInPolicy"),
-            port=3843,  # Lab defines 4 of them: [3843, 4584, 4843, 60000]
-            scopes=scopes,
+            client_id="b876a048-55a5-4fc5-9403-f5d90cb1c852", port=3843,
```

Comment:
```suggestion
            client_id="b876a048-55a5-4fc5-9403-f5d90cb1c852",
            port=3843,  # Lab defines 4 of them: [3843, 4584, 4843, 60000]
```

Suggests to keep the original inline comment. Back then, it took us a while to find those info from the doc ocean. And, if without such "justification", it would be weird to understand where that magic number "3843" came from.